sa/sa.go

```diff
@@ -1759,10 +1760,19 @@ func (ssa *SQLStorageAuthority) FinalizeAuthorization2(ctx context.Context, req
 // RevokeCertificate stores revocation information about a certificate. It will only store this
 // information if the certificate is not already marked as revoked.
 func (ssa *SQLStorageAuthority) RevokeCertificate(ctx context.Context, req *sapb.RevokeCertificateRequest) (*emptypb.Empty, error) {
-	if req.Serial == "" || req.Date == 0 || req.Response == nil {
+	if req.Serial == "" || req.Date == 0 {
 		return nil, errIncompleteRequest
 	}
+	if req.Response == nil && !features.Enabled(features.ROCSPStage6) {
+		return nil, errIncompleteRequest
+	}
+
 	revokedDate := time.Unix(0, req.Date)
+	ocspResponse := req.Response
+	if features.Enabled(features.ROCSPStage6) {
+		ocspResponse = nil
+	}
```

Comment:
I don't believe this is necessary. We already know this is nil (and that's fine) based on the check on line `1766`.